frodo-kem/src/error.rs

```diff
@@ -0,0 +1,31 @@
+/*
+    Copyright Michael Lodder. All Rights Reserved.
+    SPDX-License-Identifier: Apache-2.0
```

Comment:
License specified here is not equal to the one in the `Cargo.toml` (license = "Apache-2.0 OR MIT")